service/src/test/java/bio/terra/externalcreds/BaseTest.java

```diff
@@ -14,6 +15,7 @@
 @ActiveProfiles({"test", "human-readable-logging"})
 @Transactional
 @Rollback
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
```

Comment:
What does this annotation do?

Reply:
It makes sure that Spring throws out its state after each test file. Without this, Spring keeps around its db connections for completed tests, and we end up running out of db connections!